README.md

```diff
@@ -19,7 +19,7 @@ It requires a configuration file. See [hyperkube-config.yaml.example](hyperkube-
 pip3 install hyper-kube-config 
 ```
 
-## Setup `~/.hyperkube-config.yaml` file
+## Setup `~/hyperkube-config.yaml` file
 
 The default locations for the config file is `~/.hyperkube-config.yaml`. You can also place the config file at a different location and pass the location as a command line option `-c <hyper-kube-config-location>` or `--config <hyper-kube-config-location>`
```

Comment:
Does the name in the paragraph below need to match the title?

Reply:
Done